pkg/auth/authproviders/openshift/internal/dexconnector/openshift.go

```diff
@@ -219,7 +235,13 @@ func (c *openshiftConnector) HandleCallback(s connector.Scopes, r *http.Request)
 		ctx = context.WithValue(r.Context(), oauth2.HTTPClient, c.httpClient)
 	}
 
-	token, err := c.oauth2Config.Exchange(ctx, q.Get("code"))
+	ri := requestinfo.FromContext(ctx)
+	redirect_uri := MakeRedirectURI(&ri, ri.HTTPRequest.URL.Path)
```

Comment:
```suggestion
	redirectURI := MakeRedirectURI(&ri, ri.HTTPRequest.URL.Path)
```

Reply:
Nit: it makes sense to give this var and the one in `LoginURL()` the same name because they denote the same things. I don't care much whether it would be `redirectURL` or `redirectURI` : )